docs/pages/admin-guides/teleport-policy/integrations/netiq.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Discover NetIQ Access Patterns with Teleport Policy
+description: Describes how to synchronize OpenTex NetIQ access patterns using Teleport Policy and Access Graph.
+---
+
+With Teleport Policy's Access Graph, you gain insights into your NetIQ organization
+structure, the resources each user can access, and the roles that grant them access.
+Access Graph provides a visual representation that enhances security and improves
+understanding of large NetIQ organizations.  
+
+Access Graph helps answer key questions such as:  
+
+- **Which resources does each user have access to?**  
+- **Which roles grant access to specific resources?**  
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product, available to **Teleport Enterprise edition** customers.  
```

Comment:
Cloud too, right?

Reply:
We now name cloud and enterprise self hosted as Enterprise edition.